jekyll/_cci2/orb-author-faq.md

```diff
@@ -0,0 +1,101 @@
+categories: [configuring-jobs]
+order: 20
+---
+
+This document describes various questions and technical issues that you may find helpful when authoring orbs.
+
+* TOC
+{:toc}
+
+
+
+## Errors Claiming Namespace Or Publishing Orbs
+
+* Question: I receive an error when attempting to claim a namespace or publish a production orb.
+
+* Answer: You may not be an organization owner/admin.
+
+Organizations may only claim a single namespace. In order to claim a namespace for an organization the authenticating user must possess owner/admin privileges over the organization. 
+
+If you do not have the proper permissions you may see an error similar to below:
+
+
+> Error: Unable to find organization YOUR_ORG_NAME of vcs-type GITHUB: Must have member permission.: the organization 'YOUR_ORG_NAME' under 'GITHUB' VCS-type does not exist. Did you misspell the organization or VCS?
+
+
+Read more in our [Orbs Quickstart]({{site.baseurl}}/2.0/orb-author/#orbs-quickstart).
+
+
+## Secure API Tokens
+
+* Question: How do I protect a user's API tokens and other sensitive information?
+
+* Answer: Utilize the "env_var_name" parameter type for the API key parameter. This parameter type will only accept POSIX valid environment variable name strings as valid. In the parameter description it is best to mention to the user to add this environment variable. 
+
+Read more:
+* [Environment Variable Name]({{site.baseurl}}/2.0/reusing-config/#environment-variable-name)
+* [Best Practices]({{site.baseurl}}/2.0/orbs-best-practices/)
+
+## Environment Variables
+
+* Question: How can I require a user to add an environment variable?
+* Answer: Create a parameter for the environment variable name, even if it is a statically named environment variable the user _should not_ change, and assign it the correct default value. In the parameter description let the user know if this value should not be changed. In either event instruct the user where they can obtain their API key. 
+
+Read more:
+* [Environment Variable Name]({{site.baseurl}}/2.0/reusing-config/#environment-variable-name)
+* [Best Practices]({{site.baseurl}}/2.0/orbs-best-practices/)
+
+## Supported Programming Languages
+
+* Question: What programming language can I write my orb in?
+* Answer: Orbs are packages of [CircleCI YAML config]({{site.baseurl}}/2.0/configuration-reference/) language. CircleCI config may contain commands that are issued against a given [Executor]({{site.baseurl}}/2.0/executor-intro/).
+
+CircleCI orbs package [CircleCI reusable config]({{site.baseurl}}/2.0/reusing-config/), such as [Commands]({{site.baseurl}}/2.0/reusing-config/#authoring-reusable-commands), which can execute within a given [Executor]({{site.baseurl}}/2.0/executor-intro/) defined by either the user if using a command within a custom job, or by the orb author if using a [Reusable Job]({{site.baseurl}}/2.0/orb-author-intro/#jobs). The environment within which your logic is running may influence your language decisions.
```

Comment:
I find the first line on `58` pretty similar to the ending of `line 56` ("CircleCI config may contain commands that are issued against a given [Executor]({{site.baseurl}}/2.0/executor-intro/).")

Can we remove one of these?

Reply:
Removed from line 56.